cas-management-webapp-support/src/main/java/org/jasig/cas/config/CasManagementWebAppConfiguration.java

```diff
@@ -43,6 +43,7 @@
  * @since 4.3.0
  */
 @Configuration("casManagementWebAppConfiguration")
+@Lazy(true)
```

Comment:
OK. And it makes a big difference, I mean, you can see it looking a the "startup time"?